sql/4_1_2-to-4_2_0_upgrade.sql

```diff
@@ -2972,3 +2972,9 @@ ALTER TABLE `layout_options` ADD COLUMN `source` char(1) NOT NULL default 'F'
 ALTER TABLE `layout_options` ADD COLUMN
   `conditions` text NOT NULL DEFAULT '' COMMENT 'serialized array of skip conditions';
 #EndIf
+
```

Comment:
yes, should be done in 4_3_0-to-4_3_1_upgrade.sql
Also:
make billing_note a TEXT column 
After migrating the genericval2 data to billing_note, then would empty the genericval2 column.
Where is the gui to add the billing_note? Does it need to be added to the demographics layout?